README.md

```diff
@@ -252,7 +252,7 @@ ActiveAdmin.register Page do
     actions
   end
 
-  form do |f|
+  form html: { class: "edit_atomic_content" } do |f|
```

Comment:
Is the convention for DOM classes to have hyphens instead of snakes like `edit-atomic-content`?